src/main/kotlin/hu/kirdev/schpincer/service/ItemService.kt

```diff
@@ -39,6 +42,8 @@ open class ItemService {
     @Transactional(readOnly = false)
     open fun save(itemEntity: ItemEntity) {
         repo.save(itemEntity)
+        // Regenerate extras whenever an item gets updated
```

Comment:
use a better function name instead of a comment

Reply:
I like the function name 😠